app/scripts/views/sign_up.js

```diff
@@ -226,6 +232,22 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       return this._isMailcheckEnabledValue;
     },
 
+    _checkPasswordStrength: function () {
```

Comment:
Part of https://github.com/mozilla/fxa-content-server/pull/2810/files#r36750995
We need an event of how many people were enrolled into the test